README.md

```diff
@@ -321,6 +331,15 @@ client.getUnspentTransactionOutputs([{
 }], { extension: 'json' }, [callback])
 ```
 
+### command(command, [arguments...])
+Allows you to directly call bitcoind's [RPC methods](https://bitcoin.org/en/developer-reference#remote-procedure-calls-rpcs).
+
+##### Example
+
+```js
+client.command('getBlockHeadersByHash', '0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206');
```

Comment:
This isn't valid as the endpoint is different. `command` only forwards requests via the RPC api.